Assets/Scripts/EncounterScripts/EncounterManager.cs

```diff
@@ -0,0 +1,68 @@
+using UnityEngine;
+
+public class EncounterManager : MonoBehaviour
```

Comment:
not sure if we can really utilize this class to its potential, as most of its functionality is in the sceneGod. Maybe just call thigns in the TileEncounterTrigger